service/dap/server.go

```diff
@@ -128,6 +128,11 @@ type Server struct {
 	// sendingMu synchronizes writing to net.Conn
 	// to ensure that messages do not get interleaved
 	sendingMu sync.Mutex
+
+	runningMutex sync.Mutex
```

Comment:
runningMu and stopMu, for naming consistency.

Reply:
Done.